packages/cli/src/lib/yargs-cli.ts

```diff
@@ -115,7 +115,9 @@ export function yargsCli<T = unknown>(
 function validatePersistFormat(persist: PersistConfig) {
   try {
     if (persist.format != null) {
-      persist.format.forEach(format => formatSchema.parse(format));
+      persist.format
+        .flatMap(format => format.split(','))
```

Comment:
I couldn't get it in over coerce in the command options and the check function in `yargs-cli` is called before.